lib/matplotlib/scale.py

```diff
@@ -98,25 +98,26 @@ def __init__(self, nonpos='clip'):
         self._clip = {"clip": True, "mask": False}[nonpos]
 
     def transform_non_affine(self, a):
+        # Ignore invalid values due to nans being passed to the transform
         with np.errstate(divide="ignore", invalid="ignore"):
```

Comment:
It's not hurting anything, but I don't think the "divide" kwarg is needed, since the only division is by `np.log(self.base)`.